src/Annotator.ts

```diff
@@ -1,5 +1,5 @@
 interface Annotator {
-    start(manifestUrl: string): Promise<void>;
+    start(): Promise<void>;
```

Comment:
After talking to @aslagle, it seems like we can remove `start` from this interface.